tests/system/action/user/test_update.py

```diff
@@ -1125,11 +1567,25 @@ def test_update_strip_space(self) -> None:
             },
         )
 
+    def test_update_no_OML_set(self) -> None:
+        self.permission_setup()
+        self.set_user_groups(self.user_id, [2])
+
+        response = self.request(
+            "user.update",
+            {
+                "id": self.user_id,
+                "meeting_id": 1,
+                "group_ids": [1],
+            },
+        )
+        self.assert_status_code(response, 200)
+
     def test_update_history_user_updated_in_meeting(self) -> None:
         self.set_models(
             {
                 "user/111": {"username": "user111"},
-                "meeting/110": {"is_active_in_organization_id": 1},
+                "meeting/110": {"is_active_in_organization_id": 1, "name": "Test"},
```

Comment:
Test data should only contain necessary data for the test. All kinds of names etc. are mostly irrelevant for the functionality and should therefore not be included in the tests

Reply:
done